### PR TITLE
update location name in template

### PR DESCRIPTION
**Pull request by sandlerr**
cc @zendesk/vegemite

### CODE REVIEW

File: app_template_iframe/manifest.json.tt [hunk 1]
```diff
@@ -7,7 +7,7 @@
   },
   "defaultLocale": "en",
   "private": true,
-  "location": { "zendesk":
```
Comment (danielbreves) [inline app_template_iframe/manifest.json.tt]: Can you also fix the indentation while you're here please? 😬

Comment (sandlerr) [inline app_template_iframe/manifest.json.tt]: I'm not sure what's wrong with it? Put this on the next line?

Comment (danielbreves) [inline app_template_iframe/manifest.json.tt]: Yea, I think it should be like this:
```json
{
  "location": {
    "support": {
      "ticket_sidebar": "<%= @iframe_location %>"
    }
  }
}
```

Comment (sandlerr) [inline app_template_iframe/manifest.json.tt]: @danielbreves 💇‍♂️

File: features/new.feature [hunk 2]
```diff
@@ -109,8 +109,10 @@ Feature: create a template for a new zendesk app
  },
  "defaultLocale": "en",
  "private": true,
- "location": { "zendesk":
-   { "ticket_sidebar": "assets/iframe.html" }
+ "location": {
+   "support": {
+    "ticket_sidebar": "assets/iframe.html"
```
Comment (danielbreves) [inline features/new.feature]: Missing 1 space character here 😸

Comment (sandlerr) [inline features/new.feature]: done, feel free to implement a linter and add it to the travis run if this kinda thing bothers you

**Pull request merged**